itest/test/spectre/spectre_test.py

```diff
@@ -747,11 +747,10 @@ def test_gzip_is_disabled(self):
         assert body1 == body2
         assert body2 == body3
 
-    def test_msgpack_is_disabled(self):
+    def test_msgpack_is_allowed_for_normal_endpoints(self):
         resp = get_through_spectre('/timestamp/get', extra_headers={'Accept': 'application/msgpack'})
         assert resp.headers['Spectre-Cache-Status'] == 'miss'
-        assert 'Accept' not in resp.json()['received_headers']
-        assert 'accept' not in resp.json()['received_headers']
+        assert 'accept' in resp.json()['received_headers']
```

Comment:
Assert against the value of the accept header as well, it'll be a stronger guarantee: `assert resp.json()['received_headers']['accept'] = 'application/msgpack'`